File: Hospital/Models/Pacientes.cs
namespace Hospital.Models
{
    public class Pacientes : Usuarios
    {
        public string NumSeguridadSocial { get; set; }
        public string NumTarjeta { get; set; }
        public string Telefono { get; set; }
        public string Direccion { get; set; }
        public virtual ICollection<Citas> Citas { get; set; }
    }

}

File: App_medica/Citas_Medicas_API/Citas_Medicas_API/src/app/API/models/pacientes.ts
/* tslint:disable */
/* eslint-disable */
import { Citas } from './citas';
export interface Pacientes {
  apellidos?: null | string;
  citas?: null | Array<Citas>;
  clave?: null | string;
  direccion?: null | string;
  nombre?: null | string;
  numSeguridadSocial?: null | string;
  numTarjeta?: null | string;
  telefono?: null | string;
  usuario?: null | string;
}
